dvc/main.py

```diff
@@ -53,6 +54,15 @@ def main(argv=None):
         ret = 253
     except DvcParserError:
         ret = 254
+    except UnicodeError:
+        if is_py2:
```

Comment:
So you'll print nothing (not even unexpected error) on py3 if UnicodeError is caught. This is wrong.